afdko/Tools/SharedData/FDKScripts/BezTools.py

```diff
@@ -1298,7 +1297,7 @@ def getFontInfo(self, fontPSName, inputPath, allow_no_blues, noFlex, vCounterGly
 		for i in range(3, numBlueValues,2):
 			blueValues[i] = blueValues[i] - blueValues[i-1]
 
-		blueValues = map(str, blueValues)
+		blueValues = list(map(str, blueValues))
```

Comment:
`[str(v) for v in blueValues]` is more idiomatic

Reply:
see c559f33b4b1c3aa8f3768eb15159edec5ea55134